Fenced_Frames_Ads_Reporting.md

```diff
@@ -183,3 +195,24 @@ function addBeaconData(element) {
 ```
 
 The beacon data will be in place by the time that the navigation starts. When the navigation commits, the automatic beacon will be sent out with event data set to "link1 was clicked.".
+
+# Support for Ad Components
+## Goal
+When a rendered ad is composed of [multiple pieces](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#34-ads-composed-of-multiple-pieces), it is useful to detect user clicks that happened on ad components.
+
+## Design
+### Event Type and Reporting Destination
+For fenced frames rendering the ad components under the top-level ad fenced frame, the `reserved.top_navigation` event type and corresponding reporting destination registered for the top-level fenced frame are reused when beacons are sent from the ad component fenced frames.
+
+### Restricted to send `reserved.top_navigation` beacons only
+* Invocation of the `reportEvent` API from an ad component fenced frame is disallowed.
+* The only supported beacon to be sent from an ad component fenced frame is the `reserved.top_navigation` automatic beacon. Note this beacon is gated on a user activation (e.g. click).
+* To ensure that there is no arbitrary data that can be received at the server from the component ad, the `eventData` field via `window.fence.setReportEventDataForAutomaticBeacons`, if specified, will be ignored.
```

Comment:
Add the privacy rationale: "This is to ensure that the component ad url is not revealed in the event report, or else it could lead to the join of 2 k-anonymous urls (parent and component ad) at the receiving server."

@JensenPaul does this sg to you?

Reply:
A little rewrite: "This ensures that information from the component ad URL is not revealed in the event report, or else it could lead to the join of two independently k-anonymous URLs (parent and component ad) at the receiving server."